lib/FrontendTool/MakeStyleDependencies.cpp

```diff
@@ -132,6 +132,12 @@ bool swift::emitMakeDependenciesIfNeeded(DiagnosticEngine &diags,
     dependencyString.push_back(' ');
     dependencyString.append(frontend::utils::escapeForMake(path, buffer).str());
   }
+  auto macroPluginDependencyPath =
+      reversePathSortedFilenames(depTracker->getIncrementalDependencyPaths());
+  for (auto const &path : macroPluginDependencyPath) {
+    dependencyString.push_back(' ');
+    dependencyString.append(frontend::utils::escapeForMake(path, buffer).str());
+  }
```

Comment:
Good catch. I don't see a test for this, though?

Reply:
Actually, this has an error `getIncrementalDependencyPaths()` -> `getMacroPluginDependencyPaths()`. Thanks for pointing it out!